pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperKits/JuniperKitIntegration.java

```diff
@@ -0,0 +1,61 @@
+package org.broadinstitute.dsm.juniperKits;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.atomic.AtomicReference;
+
+import com.typesafe.config.Config;
+import org.broadinstitute.dsm.DSMServer;
+import org.broadinstitute.dsm.TestHelper;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.KitRequestShipping;
+import org.broadinstitute.dsm.model.JuniperKitRequest;
+import org.broadinstitute.dsm.model.KitRequest;
+import org.broadinstitute.dsm.model.KitUploadObject;
+import org.broadinstitute.dsm.route.JuniperShipKitRoute;
+import org.broadinstitute.dsm.statics.ApplicationConfigConstants;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class JuniperKitIntegration {
```

Comment:
Seems like our convention is to include the term `Test` on all tests?
A nit, but I'm not sure the term `Integration` adds much here since we typically don't label tests that way?

Reply:
Good point, how about JuniperKitCreationTest?